import-export-cli/impl/importThrottlePolicy.go

```diff
@@ -41,7 +43,15 @@ func ImportThrottlingPolicyToEnv(accessOAuthToken, importEnvironment, importPath
 }
 
 func importThrottlingPolicy(endpoint string, importPath string, accessToken string, isOauth bool, ThrottlePolicyUpdate bool) error {
-	resp, err := executeThrottlingPolicyUploadRequest(endpoint, importPath, ThrottlePolicyUpdate, accessToken, isOauth)
+	exportDirectory := filepath.Join(utils.ExportDirectory, utils.ExportedPoliciesDirName, utils.ExportedThrottlePoliciesDirName)
+	resolvedPolicyFilePath, err := resolvePolicyImportFilePath(importPath, exportDirectory)
+	if err != nil {
+		return err
+	}
+
+	utils.Logln(utils.LogPrefixInfo+"Policy Location:", resolvedPolicyFilePath)
```

Comment:
```suggestion
	utils.Logln(utils.LogPrefixInfo + "Policy Location: ", resolvedPolicyFilePath)
```
Shall we fix the formatting?